plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -212,22 +212,16 @@ public class FeedReader.ttrssAPI : GLib.Object {
 					{
 						var feed_node = response.get_object_element(i);
 						string feed_id = feed_node.get_int_member("id").to_string();
-
-						bool hasIcon = feed_node.get_boolean_member("has_icon");
-
-						if(hasIcon)
-							Utils.downloadIcon(feed_id, m_iconDir + feed_id + ".ico");
-						else
-							hasIcon = Utils.downloadFavIcon(feed_id, feed_node.get_string_member("feed_url"));
+						string? icon_url = feed_node.get_boolean_member("has_icon") ? m_iconDir+feed_id+".ico" : null;
```

Comment:
Is this going to work, since it's pointing at a local file? I don't know how TTRSS works